README.md

```diff
@@ -152,6 +152,17 @@ To push without running the pre-push hook (e.g. doc updates):
 git push <remote> --no-verify
 ```
 
+Note: If while pushing you encounter this error "Could not initialize class org.codehaus.groovy.runtime.InvokerHelper" then downgrade your java version to java11
```

Comment:
I would mention that this is an issue that some people have experienced with Java14 and that downgrading to a version of Java13 or lower resolves the issue.